_security-analytics/api-tools/threat-intel/threat-intel-source.md

```diff
@@ -0,0 +1,637 @@
+---
+layout: default
+title: Threat Intelligence Source Configuration APIs
+parent: API tools
+nav_order: 50
+---
+
+# Threat Intelligence Source Configuration APIs
+
+The following APIs can be used for tasks related to threat intelligence source configurations.
```

Comment:
@eirsep  Would it be helpful to define what a `source` is for our purposes?

Reply:
Started off with the APIs
Once that's done, similar to detectors, I will define entities and detail them out in another commit to this PR

Reply:
added a page for Threat intel overview plz check